lint/src/test/java/permissions/dispatcher/CallOnRequestPermissionsResultDetectorTest.java

```diff
@@ -96,4 +97,45 @@ public void callOnRequestPermissionsResultDetector() throws Exception {
                 .expectErrorCount(1)
                 .expectWarningCount(0);
     }
+
+    @Test
+    public void callOnRequestPermissionsResultDetectorNoErrorForKotlin() throws Exception {
+        @Language("JAVA") String runtimePerms = getRuntimePermission();
+
+        @Language("JAVA") String onNeeds = getOnNeedsPermission();
+
+        @Language("JAVA") String onShow = getOnRationaleAnnotation();
+
+        @Language("kotlin") String foo = ""
+                + "package permissions.dispatcher;\n"
+                + "@RuntimePermissions\n"
+                + "class Foo : android.app.Activity {\n"
+                + "fun onRequestPermissionsResult(requestCode: Int, permissions: Array<String>, grantResults: IntArray) {\n"
+                + "super.onRequestPermissionsResult(requestCode, permissions, grantResults);\n"
```

Comment:
Nit: should we remove semicolon? 👀 line below is the same 🙇

Reply:
Nice catch! I just replaced Java to Kotlin without checking these points! Let me fix these